sdk/keyvault/azure-security-keyvault-certificates/src/certificate_client_operations.cpp

```diff
@@ -39,6 +39,11 @@ std::unique_ptr<Azure::Core::Http::RawResponse> CreateCertificateOperation::Poll
   {
     rawResponse = std::move(error.RawResponse);
   }
+  catch (...)
+  {
+    m_status = Azure::Core::OperationStatus::Running;
+    return rawResponse;
+  }
```

Comment:
I don't think we want to catch any other exception and resolve it to `running`.  What if it is an authentication issue?  or something else?  Why are you adding this for?